hooks/use-simple-transfer.ts

```
@@ -320,11 +320,13 @@ export default function useSimpleTransfer({
 
       for (const action of actions) {
         logger.debug(`Sending transaction ${action.name}:`, action);
+        
```

ESLint

[error] Delete `········`

(prettier/prettier)